pyboy/utils.py

```diff
@@ -243,16 +224,17 @@ class WindowEvent:
     ) = range(42)
 
     def __init__(self, event):
-        self.event = event
+        self.__event = event
 
-    def __eq__(self, x):
-        if isinstance(x, int):
-            return self.event == x
-        else:
-            return self.event == x.event
+    def __eq__(self, other):
+        if isinstance(other, WindowEvent):
+            return self.__event == other.__event
+        elif isinstance(other, int):
+            return self.__event == other
+        return NotImplemented
```

Comment:
I see it's correct to do `return NotImplemented`. Can you write a test case that demonstrates it?

Reply:
Possibly, add a test case that demonstrates that not only `a == b` works, but also `b == a`. This is probably done at the same time.